addons/project/report/project_report.py

```diff
@@ -65,6 +65,8 @@ class ReportProjectTaskUser(models.Model):
         column2='task_id', string='Block', readonly=True,
         domain="[('allow_task_dependencies', '=', True), ('id', '!=', id)]")
     description = fields.Text(readonly=True)
+    # We exclude template tasks, but we still need the field for the views
+    is_template = fields.Boolean(readonly=True)
```

Comment:
We should not keep an useless field in the report just because we have some generic views with `project.task` model. 😇 

If we keep it the user would be able to select it in the search view and will see nothing will happen since we already remove the tasks template in the main query. 😕